client/containers/DashboardSettings/PubSettings/Doi.tsx

```diff
@@ -243,8 +243,7 @@ class Doi extends Component<Props, State> {
 			canIssueDoi &&
 			// a deposit has not been submitted yet for this work
 			!(justSetDoi || pubData.crossrefDepositRecordId) &&
-			// the Pub is not a supplement to another work
```

Comment:
This duplicates a comment further down in the file where this check actually occurs.